### ISSUE TITLE
Network Overview Loading Indefinitely 

### ISSUE THREAD

**Issue by MACCAMCGOO** [jokob-sk/NetAlertX#905]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues and I checked the docs https://github.com/jokob-sk/NetAlertX/tree/main/docs

### The issue occurs in the following browsers. Select at least 2.

- [X] Firefox
- [X] Chrome
- [ ] Edge
- [ ] Safari (unsupported) - PRs welcome
- [ ] N/A - This is an issue with the backend

### Current Behavior

As per the bug report #608 after updating to latest release v24.10.31  yesterday I am experiencing the behaviour that the Network Overview indefinitely shows loading. 

I down graded to 24.10.12 and immediately the network map displayed.

I noticed one console error which appears when the page in initially loaded... Error: Mismatched anonymous define() module: Full message is in the attached Error 1.... there is a second error relating to network.php:2940 Uncaught TypeError: a.name.localeCompare is not a function which I have included in attachment 2. 
[NetaleterConsoleError2.txt](https://github.com/user-attachments/files/18010376/NetaleterConsoleError2.txt)
[NetAlterConsoleError1.txt](https://github.com/user-attachments/files/18010379/NetAlterConsoleError1.txt)

### Expected Behavior

Network map to appear

### docker-compose.yml

```python
#version: "3"
services:
  pialert:
    container_name: pialert
# Network Map working   image: "jokobsk/pi.alert:24.10.12"
    image: "jokobsk/pi.alert:latest"
    network_mode: "host"
    restart: unless-stopped
    volumes:
      - /home/ian/pialert/config:/app/config
      - /home/ian/pialert/db:/app/db
      - /home/ian/pialert/logs:/app/front/log
    environment:
      - TZ=Europe/London
      - PORT=20211
```

### What branch are you running?

Production

### Debug enabled

- [X] I have read and followed the steps in the wiki link above and provided the required debug logs and the log section covers the time when the issue occurs.

**Comment by jokob-sk** [comment]
Hi @MACCAMCGOO ,

Thanks for the details!

Can you please:

1. Remove any recursive assignment (e.g. assigning the Internet node to another node - this causes an infinite loop - if taht is teh case, please let me know how you did this)
2. Make sure you refresh your browser cache 
3. click the 🔄 refresh button in the top right corner.

Thanks in advance,
j

**Comment by MACCAMCGOO** [comment]
Hi J, I have got the latest code started working .... what I did was change my internet connection from "gateway" [which is what the connection was automatically detected as] to "Router". Removed the main switch connected to the gateway, then added back. Upgraded to latest version, network then showed. Changed the type back to gateway and the network map remained.
Will continue to monitor over the coming couple of days
